LICENSE.TXT

```diff
@@ -0,0 +1,279 @@
+==============================================================================
```

Comment:
instead of copying the license could you add this to the sourcebuild props:
`<PackageLicenseFile>$(RepoRoot)/llvm/LICENSE.TXT</PackageLicenseFile>`

though like I mentioned in https://github.com/dotnet/llvm-project/pull/242#discussion_r1102876858 I'm still curious why we need that.

Reply:
I had tried that, but Arcade didn't like it.

From the errors I've seen, I think this gets used by some some intermediate packages that source-build creates.

Reply:
Hmm okay, thanks for trying. It sounds like something that should be fixed so I'd prefer having an issue tracking that.

Reply:
I hope @MichaelSimons takes a look as part of the second item on https://github.com/dotnet/llvm-project/pull/439#issuecomment-1598586327.
If we can't fix it for this PR (maybe I was doing it wrong), we should create an issue.